stylesheets/index.css

```diff
@@ -0,0 +1,10 @@
+.content {
```

Comment:
I think you can combine your index.css and style.css they seem to be serving the same purpose here and then use your specific css files for the corresponding html page like you are doing.